proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/ecies/ECIESEncryptorId.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2018 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.security.powerauth.ecies;
+
+import io.getlime.security.powerauth.core.ECIESEncryptorScope;
+
+/**
+ The <code>ECIESEncryptorId</code> enumeration defines various configurations
+ for our ECIES scheme, used in the PowerAuth protocol.
+ */
+public enum ECIESEncryptorId {
+
+    /**
+     Constant for "no-encryption".
+     */
+    None(ECIESEncryptorScope.Application, null, false),
+
+    //
+    // Available for application
+    //
+
+    /**
+     Defines the encryptor for an application scope, which can be used for an application's custom
+     purposes. The application server can typically decrypt data, encrypted with this configuration.
+     */
+    GenericApplicationScope(ECIESEncryptorScope.Application, "/pa/generic/application", true),
```

Comment:
Enums in Java should use all caps, see: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Thanks, I'll also change that to numeric constant, because passing enum to / from JNI is quite heavy.